<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="container_label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>68</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Name :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="group_name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Group name</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="container_label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>68</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Deal ID :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="deal_id">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Deal ID</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="container_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>68</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Container:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="container_list">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>28</height>
        </size>
       </property>
       <property name="currentText">
        <string>Optional</string>
       </property>
       <item>
        <property name="text">
         <string>Optional</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
